### DISCUSSION TITLE
Durable orchestrator stuck in pending state for around 20.30 seconds frequently when function app scales to multiple instance

### DISCUSSION THREAD

**Discussion by ashishsure** [Azure/azure-functions-durable-extension#2052] [Q&A]
I want to understand why does orchestrator gets stuck in pending state when there are multiple instance running either in consumption or in Premium Plan?

How can we fix this ?

**Comment by cgillum** [comment]
It's probably caused by queue polling delays, which is normal when your app has low load, and you scale out to multiple app instances. This behavior and potential workarounds are described in our documentation, here: https://docs.microsoft.com/azure/azure-functions/durable/durable-functions-perf-and-scale#orchestration-start-delays
